addons/mail/static/src/components/editable_text/editable_text.js

```diff
@@ -2,6 +2,8 @@ odoo.define('mail/static/src/components/editable_text/editable_text.js', functio
 'use strict';
 
 const { Component } = owl;
+const { markEventHandled } = require('mail/static/src/utils/utils.js');
```

Comment:
define below `'use strict';` with single empty line (and keep single empty line before `const { Component } = ...`)